docs/guides/unbound.md

```diff
@@ -50,7 +50,7 @@ Fortunately, both your Pi-hole as well as your recursive server will be configur
 
 ## Setting up Pi-hole as a recursive DNS server solution
 
-We will use [`unbound`](https://www.unbound.net/), a secure open source recursive DNS server primarily developed by NLnet Labs, VeriSign Inc., Nominet, and Kirei.
+We will use [`unbound`](https://nlnetlabs.nl/projects/unbound/about/), a secure open source recursive DNS server primarily developed by NLnet Labs, VeriSign Inc., Nominet, and Kirei.
```

Comment:
Although this was a temporary redirect this is where it links to from their GH repos too

Reply:
Yeah, we should probably just use `https://github.com/NLnetLabs/unbound` here